<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChromAnalyzer</class>
 <widget class="QWidget" name="ChromAnalyzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChromAnalyzer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="6">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="openChromButton">
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Open Chromatogram</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit"/>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Peak threshold</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="peakthreshold">
      <property name="maximumSize">
       <size>
        <width>67</width>
        <height>24</height>
       </size>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>0.001000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.200000000000000</double>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Window size</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="windowsize">
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="maximum">
       <number>9999999</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="3" colspan="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>380</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="analyzeButton">
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Analyze</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>509</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="tableBox">
     <property name="title">
      <string>Peak results</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="3" colspan="3">
    <widget class="QGroupBox" name="plotterBox">
     <property name="minimumSize">
      <size>
       <width>481</width>
       <height>371</height>
      </size>
     </property>
     <property name="title">
      <string>Chromatogram viewer</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>376</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="aboutButton">
     <property name="text">
      <string>About</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="quitButton">
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
